src/domain/harbors/harborDetails/HarborDetails.tsx

```diff
@@ -1,67 +1,87 @@
 import React from 'react';
-import classNames from 'classnames';
 import { useTranslation } from 'react-i18next';
 
 import Grid from '../../../common/grid/Grid';
 import ExternalLink from '../../../common/externalLink/ExternalLink';
-import LabelValuePair from '../../../common/labelValuePair/LabelValuePair';
 import { HarborData } from '../utils';
 import styles from './harborDetails.module.scss';
 import Section from '../../../common/section/Section';
+import Text from '../../../common/text/Text';
+import { formatDimension } from '../../../common/utils/format';
+import List from '../../../common/list/List';
+import ListItem from '../../../common/list/ListItem';
 
 type Props = {
+  maps: HarborData['maps'];
   imageFile: HarborData['imageFile'];
   streetAddress: HarborData['streetAddress'];
   zipCode: HarborData['zipCode'];
   municipality: HarborData['municipality'];
-  wwwUrl: HarborData['wwwUrl'];
   servicemapId: HarborData['servicemapId'];
   maxWidth: HarborData['maxWidth'];
 };
 
 const HarborDetails: React.SFC<Props> = ({
+  maps,
   imageFile,
   streetAddress,
   zipCode,
   municipality,
-  wwwUrl,
   servicemapId,
   maxWidth,
 }) => {
-  const { t } = useTranslation();
-  const address = `${streetAddress} ${zipCode} ${municipality}`;
+  const { t, i18n } = useTranslation();
+  const address = `${streetAddress}, ${zipCode} ${municipality}`;
   const imageSrc = imageFile ? imageFile : '';
-  const url = wwwUrl ? wwwUrl : '';
   const serviceMapUrl = `${process.env.REACT_APP_SERVICE_MAP_URI}${servicemapId}`;
 
   return (
-    <Grid colsCount={3}>
-      <div className={classNames(styles.section, styles.harborAddress)}>
+    <Grid colsCount={4}>
```

Comment:
Harbor details story book `http://localhost:6006/?path=/story/harbordetails--harbor-details` seems to be broken (gives error)